.github/workflows/gradle.yml

```diff
@@ -168,7 +168,7 @@ jobs:
             ./gradlew :vividus-tests:runStories -x testVividusInitialization \
                                                 -Pvividus.configuration.environments=system/mobile_app/saucelabs/ios \
                                                 -Pvividus.configuration.suite=system/mobile_app \
-                                                -Pvividus.configuration.profiles=mobile_app/ios,saucelabs/mobile_app \
+                                                -Pvividus.configuration.profiles=saucelabs/mobile_app,mobile_app/ios \
```

Comment:
shouldn't we use the same approach for BrowserStack?

Reply:
for BS the order doesn't matter